internal/app/go-filecoin/internal/submodule/network_submodule.go

```diff
@@ -137,10 +136,9 @@ func NewNetworkSubmodule(ctx context.Context, config networkConfig, repo network
 
 	// set up graphsync
 	graphsyncNetwork := gsnet.NewFromLibp2pHost(peerHost)
-	bridge := ipldbridge.NewIPLDBridge()
 	loader := gsstoreutil.LoaderForBlockstore(blockstore.Blockstore)
 	storer := gsstoreutil.StorerForBlockstore(blockstore.Blockstore)
-	gsync := graphsyncimpl.New(ctx, graphsyncNetwork, bridge, loader, storer)
+	gsync := graphsyncimpl.New(ctx, graphsyncNetwork, loader, storer, graphsyncimpl.RejectAllRequestsByDefault())
```

Comment:
Does RejectAllRequestsByDefault still allow the block and message queries we need?